src/libraries/Common/src/Internal/Cryptography/HashProvider.cs

```diff
@@ -38,6 +38,20 @@ public void AppendHashData(byte[] data, int offset, int count)
 
         public abstract bool TryFinalizeHashAndReset(Span<byte> destination, out int bytesWritten);
 
+        // Compute the hash based on the appeneded data and doesn't reset the hash provider to allow for continued hashing
+        public virtual int GetCurrentHash(Span<byte> destination)
```

Comment:
No implementation will override this, make it `internal int GetCurrentHash(Span<byte> destination)`